Gemfile.lock

```diff
@@ -65,39 +65,39 @@ GEM
     archive-zip (0.12.0)
       io-like (~> 0.3.0)
     ast (2.4.2)
-    better_errors (2.7.1)
+    better_errors (2.9.1)
```

Comment:
There was a conflict between the Gemfile.lock on Main and this one.
I know this is slightly dramatic, but I figured it'd just be best to rebase, delete ours and then re-bundle install.

Along the way we seem to have had updates. Most are minor with the exception of `regexp_parser`.
And tests continue to pass afterwards.

Any objections to bundling like this?
I can separate out into a prior commit or reduce the number of updates if folks prefer/

Reply:
That's what I've done in previous PRs too. As long as nothing breaks, I don't see any problem with that approach.

Reply:
getting very odd errors on CI with it saying it can't find rake to precompile assets... 😬 
I'm going to make changes here more minimal to isolate what's going on.